### PR TITLE
CUMULUS-2357: Update data-migration1 to allow script access to rules/async_operations

### PR DESCRIPTION
**Pull request by Jkovarik**
**Summary:** Fix data-migration1 configuration

Addresses [CUMULUS-2357: Develop amazing new feature](https://bugs.earthdata.nasa.gov/browse/CUMULUS-2357)

## Changes

* Update lambdas/migration1 script to allow access/migration of async_operations and rule tables.

## PR Checklist

- [ ] Update CHANGELOG
- [ ] Unit tests
- [x] Adhoc testing
- [ ] Integration tests

### CODE REVIEW

File: lambdas/data-migration1/main.tf [hunk 1]
```diff
@@ -99,6 +101,8 @@ resource "aws_lambda_function" "data_migration1" {
   environment {
     variables = {
       databaseCredentialSecretArn = var.rds_user_access_secret_arn
+      RulesTable = var.dynamo_tables.rules.name
+      AsyncOperationsTable = var.dynamo_tables.async_operations.name
```
Comment (botanical) [inline lambdas/data-migration1/main.tf:105]: This is definitely a nit,  but is there a reason why `RulesTable` is first and not last for alphabetical ordering? Or does it need to be first for a specific tf reason I am unaware of? 🤔

Comment (Jkovarik) [reply lambdas/data-migration1/main.tf:105]: Not at all, alpha sorting all the things is an excellent goal!

Comment (Jkovarik) [reply lambdas/data-migration1/main.tf:103]: 43a43c1c8

**Review by botanical — APPROVED** [review]
From what I can tell, lgtm

**Comment by Jkovarik** [comment]
Closing in favor of the pairing work I did with @npauzenga  on https://github.com/nasa/cumulus/pull/2029

**Pull request closed without merging**